ow_lander/config/ros_controllers.yaml

```diff
@@ -80,47 +80,54 @@ arm_controller:
     - j_dist_pitch
     - j_hand_yaw
     - j_scoop_yaw
+  constraints:
+    # If the timestamp of the goal trajectory point is t, then following the trajectory 
+    # succeeds if it reaches the goal within t +/- goal_time, and aborts otherwise.
+    goal_time: 50.0 
```

Comment:
Based on your comment this is a tolerance value and I presume it is in seconds; What is the reason you picked such a high value. Ultimately if trajectory execution takes time substantially longer than expected then we want to controller to abort the execution.

Reply:
?

Reply:
Just copying-pasting from our conversation, for the record: "So I think I opted for a high value just to be conservative in special cases. For instance sometimes it happens that the scoop, for example while digging, gets momentarily stuck in the trench and then eventually it comes out. 50s is not that long, it's not even one minute. I don't think we really need to have a strict time constraint."

Reply:
Yes, it is not a big deal, maybe once we have more experience with it we can revise the values again.